Jenkinsfile

```diff
@@ -64,27 +64,51 @@ spec:
     }
   }
 
+  environment {
+        GOOGLE_APPLICATION_CREDENTIALS    = '/home/jenkins/dev/jenkins-deploy-dev-infra.json'
+    }
+
 
   stages {
     stage('Setup access') {
       steps {
         container('k8s-node') {
-          script {
-                env.KEYFILE = "/home/jenkins/dev/jenkins-deploy-dev-infra.json"
-            }
+          // env.CLUSTER_ZONE will need to be updated to match the
```

Comment:
Need to add env variables for `USER_PASSWORD` and `PG_ADMIN_CONSOLE_PASSWORD`, otherwise scripts require user input.

Reply:
Yeah these environment variables will be set in the Jenkins job. It's outside the scope of these files.

I added teardown to the post step.

I don't really want to mess with peoples gcloud config unless I have to. If they want to turn that off they can.

Reply:
Sounds good

Reply:
@techgnosis can/should we add some validation to check that required env vars are set?  I suspect we get strange behavior if one is missed?